Notify/Notify/Notify.Android/Services/GoogleMapsService.cs

```diff
@@ -0,0 +1,154 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Android.Content;
+using Android.Content.PM;
+using Geolocation;
+using Notify.Azure.HttpClient;
+using Notify.Core;
+using Notify.Services;
+using Xamarin.Essentials;
+using Location = Xamarin.Essentials.Location;
+
+namespace Notify.Droid.Services
+{
+    public class GoogleMapsService : ExternalMapsService
+    {
+        private readonly LoggerService r_Logger = LoggerService.Instance;
+        private readonly Context r_Context;
+
+        private GoogleMapsService(Context context)
+        {
+            r_Context = context;
+        }
+
+        public static ExternalMapsService Initialize(Context context)
+        {
+            if (m_Instance == null)
+            {
+                m_Instance = new GoogleMapsService(context);
+            }
+    
+            return m_Instance;
+        }
+
+        public static GoogleMapsService GetInstance()
```

Comment:
Rename it "Instance"

Reply:
It's a problem because you need to initialize the member that is in the share code, and this class inherits from the ExternalMapsService class.